libs/service-portal/assets/src/screens/Overview/Overview.tsx

```diff
@@ -152,16 +179,7 @@ const VehiclesOverview = () => {
             <Box marginRight={2} marginBottom={[1, 1, 1, 1]}>
```

Comment:
```suggestion
            <Box marginRight={2} marginBottom={1}>
```
Isn't it unnecessary repeat the margin value if it doesn't change?